src/components/Collapsible/README.md

```diff
@@ -90,6 +91,70 @@ function CollapsibleExample() {
 }
 ```
 
+### Nested collapsible
```

Comment:
Is there a way to exclude examples from the styleguide? This isn't best practice but is important that it works.

Reply:
We'd need something like `<!-- example-for: none -->` but I don't think we have one at the moment

Reply:
Would be great if we could grab context from the admin where nested collapsible are used to provide context on valid use cases. Unfortunately, I can't remember where it's being used, only that is it 😕

Reply:
Add `../src/components/**/*.stories.tsx` to https://github.com/Shopify/polaris-react/blob/master/.storybook/main.js#L18

Then create a Collapsible.stories.tsx and dump your storybook stories in there using [the standard CSF format](https://storybook.js.org/docs/react/api/csf)